a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -206,10 +206,10 @@ <h2>General overview</h2>
 			</div>
 
 			<p>Metadata found either inside a digital publication or in a corresponding external record may have
-				important accessibility claims that helps end users find and determine if the publication can meet their
+				important accessibility claims that help end users find and determine if the publication can meet their
 				specific accessibility needs.</p>
 
-			<p>All accessibility metadata is meant to be machine-readable – except for the accessibility summary - in
+			<p>All accessibility metadata is meant to be machine-readable, except for the accessibility summary. in
 				this way accessibility metadata can be extracted and displayed uniformly across different publications
```

Comment:
A summary can be read by machine, but it's not really processable in the same way as controlled terms, so I'm not sure this is the best explanation. Maybe this could be:

Most accessibility metadata uses controlled vocabularies to allow it to be extracted and displayed uniformly across different publications and localized to different user interface languages. The one exception is the accessibility summary, which allows accessibility statements that are unique to a publication.

But I wonder if we really need to get into the technicality of the summary. Maybe the last sentence could just be dropped.